feature/policy_forwarding/otg_tests/mpls_gre_udp_macsec/README.md

```diff
@@ -22,26 +22,25 @@ Test uses aggregate 802.3ad bundled interfaces (Aggregate).
 * Send bidirectional traffic:
   * IP to Encap Traffic: The IP to Encap traffic is from ATE Ports [1,2] to ATE Ports [3,4,5,6]. 
 
-  * Encap to IP Traffic: The Encap traffic to IP traffic is from ATE Ports [3,4,5,6] to ATE Ports [1,2].  
+  * Encap to IP Traffic: The Encap traffic to IP traffic is from ATE Ports [3,4,5,6] to ATE Ports [1,2].
 
 Please refer to the MPLSoGRE [encapsulation PF-1.14](feature/policy_forwarding/otg_tests/mpls_gre_ipv4_encap_test/README.md) and [decapsulation PF-1.12](feature/policy_forwarding/otg_tests/mpls_gre_ipv4_decap_test/README.md) READMEs for additional information on the test traffic environment setup.
 
 ## PF-1.17.1: Generate DUT Configuration
 ### MACsec
 * Configure MACsec Static Connectivity Association Key (CAK) Mode on both ends of the aggregate bundle links connecting ATE ports 1,2 and DUT:
-    * Define first Policy(1) to cover must-secure scenario  
-    * Define second Policy(2) to cover should-secure scenario
+    * Define first Policy(1) to cover must-secure scenario, as defined below
+    * Define second Policy(2) to cover should-secure scenario, as defined below
     * Define 5 pre-shared keys (with overlapping time of 1 minute and lifetime of 2 minutes) for both Policy(1) and Policy(2)
     * Each pre-shared key mush have a unique Connectivity Association Key Name(CKN) and Connectivity Association Key(CAK)
-    * Set CKN as encrypted/hidden in the running configuration
+    * Set CAK as encrypted/hidden in the running configuration
     * Use 256 bit cipher GCM-AES-256-XPN and an associated 64 char CAK-CKN pair
     * Set Key server priority: 15
-    * Set Security association key rekey interval: 28800 seconds
+    * Set Security association key rekey interval: 30 seconds
```

Comment:
Do we really want to keep the SAK rekey interval as 30 seconds. Is this value coming from some security requirements ?

Reply:
Perhaps to be very clear, we can test two values.  One to ensure the re-key occurs in a short timeframe (30sec) and a second value which represents a more realistic production use case (28800sec).  In this second scenario, we don't need the test to wait for a re-key in the second scenario but rather simply confirm that the value is accepted by the DUT